.github/workflows/unit-tests.yaml

```
@@ -39,27 +39,19 @@ jobs:
-          command -v md5sum
-          command -v python
-          command -v tar
-          command -v task
+      - if: "'macos-14' == matrix.os"
+      - name: "Run unit tests"
+        env:
+          CC: "/opt/homebrew/opt/llvm@16/bin/clang"
+          CXX: "/opt/homebrew/opt/llvm@16/bin/clang++"
+        run: "task test-all"
 
+      - if: "false == ('macos-14' == matrix.os)"
```

actionlint (1.7.4)

step must run script with "run" section or run action with "uses" section

(syntax-check)